Update technical.md (#69)
hubmapconsortium/portal-docs: technical.md

<details>
<summary>HuBMAP Data Ingest</summary>
    

> -   HuBMAP HIVE is responsible for producing and managing data ingest processes and associated software in collaboration with the Data Providers. HuBMAP Data Providers are responsible for producing data and metadata in collaboration with the HIVE. These processes are rapidly evolving into scalable ingest processes.

Author: ```suggestion
-   HuBMAP HIVE is responsible for producing and managing data ingest processes and associated software in collaboration with the Data Providers. HuBMAP Data Providers are responsible for producing data and metadata in collaboration with the HIVE. These processes are rapidly evolving into scalable pipelines.
```

<details>
<summary>HuBMAP Data Validation</summary>

> -   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata. Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group. The metadata schema may be accessed in Github via links from the [metadata page](https://portal.hubmapconsortium.org/docs/metadata) and downloaded as Excel spreadsheets. These spreadsheet templates have dropdown menus of constrained input for certain metadata fields.

Reviewer: ```suggestion
-   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata.
Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group.
Metadata schemas are available [here](https://hubmapconsortium.github.io/ingest-validation-tools/),
along with Excel templates with dropdowns for data entry.
```
- Suggest adding returns here to keep the diff readable.
- Suggest getting rid of `/docs/metadata`.

> -   Data providers format their data and metadata files according to the metadata and data directory schema specifications for each assay type. Required formats for metadata field input are described in the Github page for each assay-specific metadata schema. Data providers also include the required QA/QC assessments of their data as components of the submission.

Reviewer: ```suggestion
-   Data providers format their data and metadata files according to the metadata and data directory schema specifications for each assay type. Required formats for metadata field input are described in the [Github page for each assay-specific metadata schema](https://hubmapconsortium.github.io/ingest-validation-tools/). Data providers also include the required QA/QC assessments of their data as components of the submission.
```

    
-   Data providers receive registration and validation guidance using [HuBMAP’s data submission guide (currently v1.0)](https://docs.google.com/document/d/1KR2TC2y-NIjbBRHTu0giSZATMUfPKxN_/edit) as well as [Ingest tool documentation](https://github.com/hubmapconsortium/ingest-validation-tools).
    
-   [HuBMAP validation tools](https://github.com/hubmapconsortium/ingest-validation-tools#readme) written in Python ensure data submissions conform to HuBMAP standards which are shared and documented for data providers to use to run many of HuBMAP’s checks on their own prior to submission. Other services include [Metadata submission conversion](https://github.com/hubmapconsortium/tableschema-to-template#readme), [ingest validation](https://github.com/hubmapconsortium/ingest-validation-tests#readme) and base checks (checksum, file type, etc.) as well as [assay-specific checks](https://portal.hubmapconsortium.org/docs/assays).
    
-   HuBMAP staff conduct 178 (and growing) automated and manual QA/QC checks as part of the data submission & publication process. Manual validation steps are being automated as development capacity allows.
    

> -   Prior to publication, each dataset is formally approved by the data-providing institution and one or more HIVE members. Data providers must also confirm the quality of spatial and semantic metadata using the [CCF EUI](https://portal.hubmapconsortium.org/ccf-eui).

Reviewer: Unfortunately, what precisely the approval process is still unclear.

Author: @mccalluc Peter says he and Melissa wrote the current text and that I have "broad authority" over this part of the documentation. I'll check with Peter to find out who they had review their text. I value your opinion so if you have any comments or edits, please add them.

Author: @mccalluc Excellent fixes! Thanks.

-   We also automatically collect and display [HuBMAP-generated and referenced publications](https://scholar.google.com/citations?user=CtGSN80AAAAJ&hl=en) using Google Scholar.
</details>

> 
> 
> <details>
> <summary>HuBMAP Assays and Metadata</summary>
>     
> -   HuBMAP assay documentation and links to assay metadata template are accessible from the [metadata page](https://portal.hubmapconsortium.org/docs/metadata).

Reviewer: ```suggestion
```
- Please balance tags.
- Again, suggest deleting `docs/metadata`
